js/ui/scheduler/ui.scheduler.header.js

```diff
@@ -210,6 +212,7 @@ var SchedulerHeader = Widget.inherit({
 
         this._viewSwitcher = this._createComponent($element, DropDownMenu, {
             onItemClick: this._updateCurrentView.bind(this),
+            buttonIcon: themes.isMaterial() ? "chevrondown" : "overflow",
```

Comment:
Can you make themes.isMaterial() checking in a default options config? And then use some option for such checking?